lib/trino-filesystem-manager/src/main/java/io/trino/filesystem/manager/FileSystemConfig.java

```diff
@@ -18,9 +18,9 @@
 public class FileSystemConfig
 {
     private boolean hadoopEnabled = true;
-    private boolean nativeAzureEnabled;
-    private boolean nativeS3Enabled;
-    private boolean nativeGcsEnabled;
+    private boolean nativeAzureEnabled = true;
```

Comment:
@electrum I had a quick chat with @mosabua and he mentioned that we dont want to enable these by default (additionally, also disable `hadoopEnabled`) . 

I understand we load all three cloud fs modules this way, but from product-tests executed internally I do not seem to see any conflict related issues. Can you elaborate on the plan?

Reply:
Keep in mind that required properties are different across modules and that we will have more filesystem modules. Currently in the Alluxio one is in the works.

I think the user experience for new users is actually better if they're forced to determine what file system they want to use and then get the right information about required and optional properties.

Reply:
We can definitely do this in the docs as well, making it clear in the object storage connectors that you must select and configure a file system.

Reply:
fyi @findinpath